eo-runtime/src/main/java/EOorg/EOeolang/EObytes$EOxor.java

```diff
@@ -51,16 +52,28 @@ public class EObytes$EOxor extends PhDefault {
      */
     public EObytes$EOxor(final Phi sigma) {
         super(sigma);
-        this.add("b", new AtFree());
+        this.add("b", new AtVararg());
         this.add(
             "φ",
             new AtComposite(
                 this,
-                rho -> new Data.ToPhi(
-                    new Param(rho).fromBytes(BigInteger.class)
-                        .xor(new Param(rho, "b").fromBytes(BigInteger.class))
-                        .toByteArray()
-                )
+                rho -> {
+                    BigInteger base = new Param(rho).fromBytes(BigInteger.class);
+                    final Phi[] args = new Param(rho, "b").strong(Phi[].class);
+                    for (int index = 0; index < args.length; ++index) {
+                        final Object val = new Dataized(args[index]).take();
+                        if (!(val instanceof byte[])) {
```

Comment:
@kerelape I would suggest using class `Param`, which does exactly this type checking and is used in most of our classes.

Reply:
@yegor256 using `new Param(rho).strong(byte[].class)` breaks the build, while `new Dataized(rho).take(byte[].class)` does not. If `Param` is used, then there is a 50/50 chance for some tests to get stuck forever.